js/wp-autoupdates.js

```diff
@@ -0,0 +1,202 @@
+jQuery(function ($) {
+	// Disable auto-updates for a plugin.
+	$('.autoupdates_column').on('click', 'a.plugin-autoupdate-disable', function (e) {
+		e.preventDefault();
+		var $anchor = $( this );
```

Comment:
It might make it easier to notice the change in wording while the ajax request is processing if there were a `$( this ).blur()` just after the `e.preventDefault()` ... that is, make the link lose focus while the processing is happening.

And do the same for the `on.click` handler for themes.

@audrasjb are there any accessibility concerns with losing focus there (e.g., for screen readers)?

Reply:
@pbiron @audrasjb I'll leave this open for discussion. I've added the blur/focus behaviors in and it works as expected. I've also loaded the spinning icon back in only for Ajax. Dots could be used instead as mentioned, but I believe the loading animation to be more intentional. 

![auto-updates-focus-animation](https://user-images.githubusercontent.com/636521/79521999-24a01080-8020-11ea-8625-d779b500fa23.gif)

![themes-screen](https://user-images.githubusercontent.com/636521/79522010-2bc71e80-8020-11ea-9041-6726dcb77787.gif)

Reply:
yeah, I like the spinning dashicon (and the ellipsis) while the processing is happening.

Also, I just noticed that the JS updates the counts in the `Enabled/Disabled` views.  Kudos for that, nice touch!!!

Reply:
**Note:** don't bother to update the PR for the following yet, but I don't don't think that  most of the CSS changes are necessary.  I've never used CSS animations before, so am not really familiar with browser support for it (nor the necessity for browser-specific versions of the CSS properties).

On all the screens where enabling/disabling happens, core enqueues `wp-admin/css/list-tables.css`, which already contains:

```
@keyframes rotation {
	0% {
		transform: rotate(0deg);
	}
	100% {
		transform: rotate(359deg);
	}
}
```

So, I think the only CSS that is needed to get the dashicon to spin is:

```
.dashicons-update.spin {
	animation: rotation 2s infinite linear;
}
```

Also note the `2s` duration there, which matches what is used for spinning the dashicon for shiny updates.

Just a thought.

Reply:
Great observation. I'll proceed when you give me the green light on this.

Reply:
Fixed with https://github.com/WordPress/wp-autoupdates/pull/61/commits/860f00fbd86fe1b5e02b4a16cbeec4f229606031

Results is a slower animation, but it works great.

![css-default](https://user-images.githubusercontent.com/636521/79806275-a304f700-832d-11ea-81be-449cf99ef2e0.gif)

![theme-single-site](https://user-images.githubusercontent.com/636521/79806291-a8fad800-832d-11ea-8907-d32b5c46e13d.gif)

Reply:
Looks good to me, thanks!